tests/Acme81/PSR4/AFunction.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Acme81\PSR4;
+
+use Acme81\Attribute\ParameterA;
+
+function aFunc(
+    #[ParameterA("my function parameter label")]
```

Comment:
I think we should make it more apparent in the tests that we don't want this attribute to be collected; for example, with a `$this->assertNotExists()`.